src/Microsoft.VisualStudio.Services.Agent/ProcessInvoker.cs

```diff
@@ -213,6 +222,13 @@ public sealed class ProcessInvoker : AgentService, IProcessInvoker
                 _proc.StandardInput.Dispose();
             }
 
+            if(!waitForExit)
+            {
+                //just give some time for the AgentService.exe to warm up
+                Thread.Sleep(5*1000);
```

Comment:
removing this code.

---
In reply to: [115210875](https://github.com/Microsoft/vsts-agent/pull/951#discussion_r115210875) [](ancestors = 115210875)